renderer/src/pages/wallet/Wallet.tsx

```diff
@@ -1,48 +1,58 @@
 import useWallet from 'src/hooks/StationWallet'
-import { formatFilValue, openExplorerLink, truncateString } from 'src/lib/utils'
+import { formatFilValue, openExplorerLink } from 'src/lib/utils'
 import TransactionHistory from './TransactionHistory'
-import TransferForm from './TransferForm'
+import PageShell from 'src/components/PageShell'
+import BorderedBox from 'src/components/BorderedBox'
+import Text from 'src/components/Text'
+import Address from 'src/components/Address'
+import LinkOut from 'src/assets/img/icons/link-out.svg?react'
 
 const Wallet = () => {
   const {
     walletBalance,
     stationAddress,
     stationAddress0x,
-    walletTransactions,
-    processingTransaction
+    walletTransactions
   } = useWallet()
 
   return (
-    <div>
-        <h1 className='font-bold mb-4'>Wallet</h1>
-        <div className="flex gap-8">
-            <section className="w-1/3">
-                <div className='mb-4'>
-                    <h2>Station wallet balance</h2>
-                    {formatFilValue(walletBalance)}{' '}FIL
-                </div>
-                <div className='flex justify-between'>
-                    <p>Station address</p>
-                    <button type='button' onClick={() => openExplorerLink(stationAddress)}>Explorer</button>
-                </div>
-                <div className='flex justify-between'>
-                    <span>{truncateString(stationAddress)}</span>
-                    <span>{truncateString(stationAddress0x)}</span>
-                </div>
-
-                <div className='my-4'>
-                    <TransactionHistory
-                        walletTransactions={walletTransactions}
-                        processingTransaction={processingTransaction}
-                    />
-                </div>
-            </section>
-
-            <section className="w-1/3">
-                <TransferForm />
-            </section>
-
-        </div>
+    <div className='w-full flex'>
+      <PageShell>
```

Comment:
What's the purpose of the `PageShell` component? Could it be part of `Layout`? And why does it not contain the `w-full flex` element? Just for my understanding 🙏

Reply:
Different pages of the app have different layouts (Settings content has a centered max width, Dashboard has no max width, Wallet has no max width and has that extra section on the right), but share areas with the same padding/margin. PageShell provides that uniform padding/margin. It could be part of `Layout`, if it weren't for the section on the right in the modules page, so in this case it's easier for each page to compose its own layout with `PageShell`

But `PageShell` is probably not a good name, because it implies it wraps everything. I could change it to `Container`. wdyt?

Reply:
Got it! Some suggestions:

- `Main` - since semantically it's also the `<main>` element
- `Container` - as you suggested, this is a well enough understood concept

> It could be part of Layout, if it weren't for the section on the right in the modules page

I don't want to bike shed this, but one idea: What about instead adding this to layout, and adding an option there, for the modules page?

Definitely let's go with the simplest solution here.

Reply:
> What about instead adding this to layout, and adding an option there, for the modules page?

That would add some complexity, as then each page would need to import Layout, meaning the common components inside Layout (Sidebar, DraggableArea, etc) would need to go somewhere else. 
Honestly I'm ok with just using regular CSS for now though, as there are only 4 pages, and it may be easier to figure out a good abstraction later on.

Reply:
> Honestly I'm ok with just using regular CSS for now though, as there are only 4 pages, and it may be easier to figure out a good abstraction later on.

Sounds good!